<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Array upload</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="maximumSize">
           <size>
            <width>622</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Actions</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton_save_vsm">
               <property name="text">
                <string>Save VS SampleSheet</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_up_vsm">
               <property name="text">
                <string>Import to Variant Studio</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_save_bf">
               <property name="text">
                <string>Save BFM SampleSheet</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_up_bf">
               <property name="text">
                <string>Import to Bluefuse Multi</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="maximumSize">
           <size>
            <width>622</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Data</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="label_d1">
               <property name="text">
                <string>Barcode path</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_d3_data">
               <property name="text">
                <string>Data</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label_d2_data">
               <property name="text">
                <string>Data</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_d1_data">
               <property name="text">
                <string>Data</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_d3">
               <property name="text">
                <string>Array data path</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_d2">
               <property name="text">
                <string>Manifests path</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Sample sheets path</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Project name</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Investigator</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="maximumSize">
           <size>
            <width>622</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Experiment details</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Project type:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="comboBox_project_type">
               <property name="minimumSize">
                <size>
                 <width>220</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>220</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Project name:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_project_name">
               <property name="minimumSize">
                <size>
                 <width>220</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>220</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Investigator:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_investigator">
               <property name="minimumSize">
                <size>
                 <width>211</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>211</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDateEdit" name="dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Experiment date:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="maximumSize">
           <size>
            <width>622</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Array layout details dropzone</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableWidget" name="tableWidget_dropzone">
             <property name="minimumSize">
              <size>
               <width>602</width>
               <height>295</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>602</width>
               <height>295</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="maximumSize">
           <size>
            <width>622</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Comments</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTextEdit" name="textEdit_comment">
             <property name="minimumSize">
              <size>
               <width>602</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>602</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Dragzone</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Open samples Excel sheet</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="load_samples_comboBox">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>10000</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget_XLS"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1265</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
